### PR TITLE
Fix ingress control doc related to other providers and numbering

### PR DESCRIPTION
**Pull request by ericvn**
Please provide a description for what this PR is for.

Fixes https://github.com/istio/istio.io/issues/11962

See issue for more info, but short story is that updating to the new markdown caused the numbering and indention within the ingress control page to be incorrect. This PR takes the similar text from Getting Started, which was actually copied from the ingress-control page originally, and copies it back to remove the numbering and indentation so the issues are resolved. 

Another solution would be to dig into the tabset.html to fix the actually issue.

### CODE REVIEW

**Comment by ericvn** [comment]
Before:
Screen Shot 2022-09-19 at 1 39 24 PM

After:
Screen Shot 2022-09-19 at 1 45 20 PM

File: content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md [hunk 1]
```diff
@@ -57,15 +57,14 @@ Choose the instructions corresponding to your environment:
...
-Follow these instructions if you have determined that your environment has an external load balancer.
+**Follow these instructions if you have determined that your environment has an external load balancer.**
 
 Set the ingress IP and ports:
 
 {{< text bash >}}
 $ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
 $ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].port}')
 $ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="https")].port}')
-$ export TCP_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="tcp")].port}')
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md:68]: Why are you deleting TCP_INGRESS_PORT, it's used in other docs

Comment (ericvn) [reply content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md]: Added it back.

File: content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md [hunk 2]
```diff
@@ -95,40 +94,38 @@ $ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingress
...
-Setting the ingress IP depends on the cluster provider:
+_GKE:_
 
-1.  _GKE:_
-
-    {{< text bash >}}
-    $ export INGRESS_HOST=worker-node-address
-    {{< /text >}}
+{{< text bash >}}
+$ export INGRESS_HOST=workerNodeAddress
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md:100]: why is this one camelcase, while all the rest of the instructions use dash separator for placeholders?

Comment (ericvn) [reply content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md:100]: I just copied from Getting Started for consistency between pages.

Comment (frankbu) [reply content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md:100]: Hmm, maybe it should be copied from here to getting started instead?

Comment (ericvn) [reply content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md]: Updated to dash operator on both pages for consistency.

**Review by frankbu — APPROVED** [review]

**Pull request merged**